### ISSUE TITLE
Assert for not -1 instead of not infinity in parseMathExpression

### ISSUE THREAD

**Issue by overfl0** [Fusselwurm/arma-class-parser#3]
In `parseMathExpression`, you have:
```
                var
                    posOfExpressionEnd = Math.min(
                        indexOfOrMaxInt.call(raw, chars.SEMICOLON, currentPosition),
                        indexOfOrMaxInt.call(raw, chars.CURLY_CLOSE, currentPosition),
                        indexOfOrMaxInt.call(raw, chars.COMMA, currentPosition)
                    ),
                    expression = raw.substr(currentPosition, posOfExpressionEnd - currentPosition);
                assert(posOfExpressionEnd !== -1);
```

I'm not a JavaScript developer but you probably meant to assert if `posOfExpressionEnd` is not infinity instead of -1

**Comment by Fusselwurm** [comment]
right. i suspect i 
* added the assertion first,
* then created indexOfOrMaxInt 
* forgot about changing the assertion.

**Comment by Fusselwurm** [comment]
oh, I fixed that in 76fffdc428d6bbe025e8c92f66a2cab9edfa24d5
